Content.Shared/Implants/Components/ImplanterComponent.cs

```diff
@@ -15,6 +16,19 @@ public sealed partial class ImplanterComponent : Component
     public const string ImplanterSlotId = "implanter_slot";
     public const string ImplantSlotId = "implant";
 
+    /// <summary>
+    /// Whitelist to check entities against before implanting.
+    /// Implants get their own whitelist which is checked afterwards.
+    /// </summary>
+    [DataField]
+    public EntityWhitelist? Whitelist;
+
+    /// <summary>
+    /// Blacklist to check entities against before implanting.
+    /// </summary>
+    [DataField]
+    public EntityWhitelist? Blacklist;
+
```

Comment:
Should network these for prediction.

Reply:
the entire interaction logic is serverside tho it wouldnt predict anything

Reply:
@metalgearsloth should i still do it...

Reply:
Component should be sharedTM.

Reply:
yes i added